DB/projects.json

```
@@ -3499,5 +3499,16 @@
         "tech": ["flutter", "hive", "dart"]
     }
   ]
-  }
+  },
+{
+  "github_username": "blindaks",  
+  "Projects": [
+    {
+      "github_url": "https://github.com/blindaks/Quiz-App",
+      "title": "Quiz App",
+      "description": "A quiz app based on front wnd development questions.",
+      "tech": ["HTML", "CSS", "JavaScript"]
```

convert to small case 
```suggestion
      "tech": ["html", "css", "javascript"]
```